# Logfile created on 2022-12-04 11:09:51 +1300 by logger.rb/v1.5.0
2022/12/04-11:09:51: 
Starting bot!


2022/12/04-11:28:16: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:28:16: "An error has occurred:"
"There is an issue connecting to your database with your username/password, username: root.

Please check your database configuration to ensure the username/password are valid.

- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/mysql2_adapter.rb:49:in `rescue in new_client'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/mysql2_adapter.rb:43:in `new_client'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/mysql2_adapter.rb:23:in `mysql2_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:656:in `public_send'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:656:in `new_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:700:in `checkout_new_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:679:in `try_to_checkout_new_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:640:in `acquire_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:341:in `checkout'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:181:in `connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_adapters/abstract/connection_handler.rb:211:in `retrieve_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_handling.rb:313:in `retrieve_connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/connection_handling.rb:280:in `connection'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/model_schema.rb:407:in `table_exists?'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/attribute_methods/primary_key.rb:95:in `get_primary_key'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/attribute_methods/primary_key.rb:83:in `reset_primary_key'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/attribute_methods/primary_key.rb:71:in `primary_key'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/relation/delegation.rb:93:in `primary_key'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/relation/finder_methods.rb:575:in `ordered_relation'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/relation/finder_methods.rb:541:in `find_nth_with_limit'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/relation/finder_methods.rb:534:in `find_nth'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/relation/finder_methods.rb:146:in `first'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/activerecord-7.0.4/lib/active_record/querying.rb:22:in `first'
- C:/Users/Jeremy/Documents/projects/guilty_gear_discord_bot/commands/other_commands.rb:19:in `block in set_other_commands'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/discordrb-3.4.0/lib/discordrb/commands/parser.rb:108:in `call'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/discordrb-3.4.0/lib/discordrb/commands/command_bot.rb:233:in `execute_command'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/discordrb-3.4.0/lib/discordrb/commands/command_bot.rb:333:in `simple_execute'
- C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/discordrb-3.4.0/lib/discordrb/commands/command_bot.rb:491:in `block in execute_chain'"

2022/12/04-11:29:04: 
Starting bot!


2022/12/04-11:29:14: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:21: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:22: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:29: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:32: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:35: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:36: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:29:38: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-11:38:46: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-12:44:42: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-12:44:49: text is 'Anji still sucks though' length 23 negators = 0
2022/12/04-12:44:49: score = 0.25
2022/12/04-13:13:29: text is 'anji sucks balls' length 16 negators = 0
2022/12/04-13:13:29: score = -0.25
2022/12/04-13:13:55: 
New command received:
"goodbot - message: !goodbot"

2022/12/04-13:51:12: 
Starting bot!


2022/12/04-13:51:39: 
Starting bot!


2022/12/04-13:51:46: 
Starting bot!


2022/12/04-13:52:27: 
New command received:
"gitpull - message: !gitpull gio"

2022/12/04-15:13:45: 
Starting bot!


2022/12/04-15:14:10: 
New command received:
"gitpull - message: !gitpull gio"

2022/12/04-15:14:57: 
Starting bot!


2022/12/04-15:15:20: 
New command received:
"gitpull - message: !gitpull gio"

